### ISSUE TITLE
Retire Failure techniques

### ISSUE THREAD

**Issue by hidde** [w3c/wai-wcag-supporting-documents-redesign#9]
JF: 

> While not a hill to die on, I'd like to see us look to retire "Failure Techniques" for WCAG 3.0 as they are dated, incomplete, and serve (IMHO) little real value: documenting how to fail will never be complete, and we should be teaching 'How To', not 'How Not To'.
> 
> Organizations that test against Failure Techniques today need to be educated that successfully avoiding all Failure Techniques still does not equate to "accessible" content.

**Comment by hidde** [comment]
Agreed it would be useful and sensible to retire these, but this would be AGWG scope, as it is explicitly out of scope for the redesign project undertaken by EOWG.
